src/app/feature-flag/service/feature-acknowledgement.service.ts

```diff
@@ -0,0 +1,87 @@
+import { EventEmitter, Inject, Injectable, OnDestroy } from '@angular/core';
+import { Http } from '@angular/http';
+import { ExtProfile, ExtUser, GettingStartedService } from '../../getting-started/services/getting-started.service';
+
+import { cloneDeep } from 'lodash';
+import { Logger } from 'ngx-base';
+import { WIT_API_URL } from 'ngx-fabric8-wit';
+import { AuthenticationService, UserService } from 'ngx-login-client';
+import { Observable, Subscription } from 'rxjs';
+
+/**
+ * A service to manage the user acknowledgement to dismiss the warning displayed by the experimental feature banner
+ */
+@Injectable()
+export class FeatureAcknowledgementService extends GettingStartedService implements OnDestroy {
+  subscriptions: Subscription[] = [];
+  showIconChanged = new EventEmitter();
+  constructor(
+    protected auth: AuthenticationService,
+    protected http: Http,
+    protected logger: Logger,
+    protected userService: UserService,
+    @Inject(WIT_API_URL) apiUrl: string) {
+    super(auth, http, logger, userService, apiUrl);
+  }
+
+  ngOnDestroy(): void {
+    super.ngOnDestroy();
+    this.subscriptions.forEach(sub => {
+      sub.unsubscribe();
+    });
+  }
+
+  /**
+   * Returns user toggle-setting whether to display the feature-icon or not.
+   *
+   * @returns {boolean} True if the user want to see the icon showing the experimental features.
+   */
+  getToggle(): Observable<boolean> {
+    let acknowledged: boolean = true;
+    let profile;
+    return this.userService.loggedInUser
+      .map(user => {
+        profile = cloneDeep(user) as ExtUser;
+        if (profile.attributes !== undefined) {
+          profile.attributes.contextInformation = (user as ExtUser).attributes.contextInformation || {};
+          acknowledged = Boolean(profile.attributes.contextInformation.featureAcknowledgement);
+        }
+        return acknowledged;
+      });
+  }
+
+  /**
+   * Save user user toggle-setting.
+   *
+   * @param {boolean} True if the user want to see the icon showing the experimental features.
```

Comment:
This is a minor thing. The way JSDoc works is that you need to also state the parameter name.
So it should be like this. 
`@param {boolean} value True if...`